---
title: Project Report
subtitle: A.Blomberg_15_17_project_summary
date: 2016-06-13
support_email: support@ngisweden.se
---

# Project Information

User Project Name
:   Fucus vesiculosus - DNA_300bp

NGI Project Name
:   A.Blomberg_15_17

NGI Project ID
:   P3719

NGI Facility
:   Genomics Production Stockholm

User Contact
:   [anders.blomberg@cmb.gu.se](mailto:anders.blomberg@cmb.gu.se)

NGI Application Type
:   de novo _(no best practice analysis)_

Submitted samples
:   2 samples

Ordered lanes
:   1 lane

Order Dates
:   _Order received:_ 2015-11-27, _Contract received:_ 2015-12-11, _Samples received:_ 2015-12-16, _Report date:_ 2016-06-13

UPPMAX Project ID
:   `b2011228`

UPPNEX project path
:   `/proj/b2011228/INBOX/P3719`



Minimum ordered reads
:   250M

Report generated by
:   Senthilkumar, 2016-06-13

[swedac]
 
# Methods

### Library construction
* Input: DNA
* Type: Illumina TruSeq Nano
* Option: 350bp
* Category: NeoPrep

### Sequencing
A) Clustering was done by 'cBot' and samples were sequenced on HiSeqX (HiSeq Control Software 3.3.39/RTA 2.7.1) with a 2x151 setup using 'HiSeq X SBS' chemistry. The Bcl to FastQ conversion was performed using bcl2fastq-v2.17.1.14 from the CASAVA software suite. The quality scale used is Sanger / phred33 / Illumina 1.8+.

### Data Flow
Raw sequencing data is demultiplexed and converted to FastQ on site before 
being transferred securely to [UPPMAX](http://www.uppmax.uu.se/) for delivery.

### Data Processing
To ensure that all sequenced data meets our guarantee of data quality and quantity,
a number of standardised bioinformatics quality control checks are performed before
delivery. These include checking the yield, sequence read quality and cross-sample contamination.

### Accredited workflow

Library preparation
:   [tick] Validated under ISO accreditation 17025:2005

Sequencing
:   [tick] Validated under ISO accreditation 17025:2005

Data Processing
:   [tick] Validated under ISO accreditation 17025:2005

Data Analysis
:   [cross] Not validated under ISO accreditation 17025:2005

# Sample Information

NGI ID | User ID | Mreads | >=Q30(%) | Status 
-------|---------|--------|----------|-------- 
P3719_201 | Fves11 neo | `475.11` | 93.25  | PASSED 


The table is also saved as parseable tab-separated text [file](A.Blomberg_15_17_sample_info.txt) for convenience. Below you can find an explanation of the header column used in the table.

* _NGI ID:_ Internal NGI sample indentifier
* _User ID:_ User submitted name for a sample
* _Mreads:_ Total million reads (or pairs) for a sample
* _>=Q30:_ Aggregated percentage of bases that have quality score more the Q30
* _Status:_ Sequencing status of sample based on the total reads



# Library Information

NGI ID | Index | Lib Prep | Avg. FS | Lib QC
-------|-------|----------|---------|--------
P3719_201 | `ND006` | A | 350 | PASSED


The table is also saved as parseable tab-separated text [file](A.Blomberg_15_17_library_info.txt) for convenience. Below you can find an explanation of the header column used in the table.

* _NGI ID:_ Internal NGI sample indentifier
* _Index:_ Barcode sequence used for the sample
* _Lib Prep:_ NGI library indentifier
* _Avg. FS:_ Average fragment size of the library
* _Lib QC:_ Reception control library quality control step status




# Lanes Information

Date | Flowcell | Lane | Clusters(M) | PhiX | >=Q30(%) | Method
-----|----------|------|-------------|------|----------|--------
160513 | `AHLT3LCCXX` | 1 | 494 | 0.45 | 93.0 | A


The table is also saved as parseable tab-separated text [file](A.Blomberg_15_17_lanes_info.txt) for convenience. Below you can find an explanation of the header column used in the table.

* _Date:_ Date of sequencing
* _Flowcell:_ Flowcell identifier
* _Lane:_ Flowcell lane number
* _Clusters:_ Number of clusters that passed the read filters (millions)
* _>=Q30:_ Aggregated percentage of bases that have a quality score of more than Q30
* _PhiX:_ Average PhiX error rate for the lane
* _Method:_ Sequencing method used. See above for description




# Aborted/Not Sequenced samples

NGI ID | User ID | Status
-------|---------|-------
P3719_101 | Fves11 | Aborted


# General Information

## Naming conventions

The data is delivered in FastQ format using Illumina 1.8 quality scores.
There will be one file for the forward reads and one file for the
reverse reads (if the run was a paired-end run).

The naming of the files follow the convention:

```
[NGI-NAME]_[BCL-CONVERSION-ID]_[LANE]_[READ]_[VOLUME].fastq.gz
```

* _NGI-NAME:_ Internal NGI sample indentifier
* _BCL-CONVERSION-ID:_ Indentifier set by bcl2fastq tool while demultiplexing
* _LANE:_ Sequencing lane that the file originates from
* _READ:_ Forward(1) or reverse(2) read indentifier
* _VOLUME:_ Volume index when file is large enough to be split into volumes

## Data access at UPPMAX

Data from the sequencing will be uploaded to the UPPNEX (UPPMAX Next
Generation sequence Cluster Storage, [uppmax.uu.se](http://www.uppmax.uu.se)),
from which the user can access it. You can find the data in the INBOX folder of the
UPPNEX project, which was created for you when your order was placed: 

```
/proj/b2011228/INBOX/P3719
```

If you have problems accessing your data, please contact NGI
[support@ngisweden.se](mailto:support@ngisweden.se).
If you have questions regarding UPPNEX, please contact
[support@uppmax.uu.se](mailto:support@uppmax.uu.se).

## Acknowledgements

In publications based on data from the work covered by this agreement,
the authors must acknowledge SciLifeLab, NGI and Uppmax:

> The authors would like to acknowledge support from Science for Life Laboratory,
> the National Genomics Infrastructure, NGI, and Uppmax for providing
> assistance in massive parallel sequencing and computational infrastructure.

# Further Help
If you have any queries, please get in touch at
[support@ngisweden.se](mailto:support@ngisweden.se).
